mlir/include/mlir-c/Dialect/LLVM.h

```diff
@@ -234,10 +234,14 @@ MLIR_CAPI_EXPORTED MlirAttribute mlirLLVMDIBasicTypeAttrGet(
     MlirContext ctx, unsigned int tag, MlirAttribute name, uint64_t sizeInBits,
     MlirLLVMTypeEncoding encoding);
 
+/// Creates a self-referencing LLVM DICompositeType attribute.
+MLIR_CAPI_EXPORTED MlirAttribute
```

Comment:
mlirLLVMDICompositeTypeAttrGetRecSelf and mlirLLVMDISubprogramAttrGetRecSelf have been missing the MLIR_CAPI_EXPORTED prefix.